apps/igloo-kit-cli/src/cli/mod.rs

```diff
@@ -92,6 +92,12 @@ async fn top_command_handler(term: &mut CommandTerminal, config: Config, command
         cluster_network: PANDA_NETWORK.to_owned(),
     };
 
+    let redpanda_config: RedpandaConfig = RedpandaConfig {
+        broker: "localhost:19092",
```

Comment:
We should make this configurable as well in a follow up PR.

We could use the config file I set up